Project.toml

```diff
@@ -35,6 +35,9 @@ ChainRulesCore = "1.13"
 EnzymeCore = "0.5, 0.6"
 GPUArraysCore = "0.1"
 KernelAbstractions = "0.9.2"
+LinearAlgebra = "<0.0.1, 1"
```

Comment:
Why `<0.0.1`? Shouldn't it be just `1` for all base libraries?

Reply:
Or just `1.9`, since this is the minimal version of Julia NNlib supports.

Reply:
I thought the `<0.0.1` was required to make all Julia versions before 1.10 work? For the upper bound I'm not sure, the guidance on this was never very clear.

Reply:
Ah, I see

Reply:
`<0.0.1` is from my attempts to make the CI happy, which failed because of some incompatibility issue of Random.jl (at least the error message suggested that). I first thought my change in this PR caused that error, but I discovered it was there before the change! So, it might be okay to remove it if the package seems to work well in your environment.